README.md

```diff
@@ -1,15 +1,57 @@
 # Overview
 
-This repository hosts test files and examples for the BuildingSync schema.  As newer versions of the schema are released, this repository will be updated to include relevant changes.  Currently, all example files are based on [schema2.0.0](https://github.com/BuildingSync/schema/releases/tag/v2.0).
+This repo contains a collection of example BuildingSync files and tools for writing and validating BuildingSync use cases as schematron files.
 
-## System Requirements 
+## Command line validation
+### Setup
+Python 3 must be installed before continuing. Check your version with `python --version`.
+```bash
+# Copy repo
+git clone https://github.com/BuildingSync/TestSuite.git
+
+# Install dependencies
+cd TestSuite
+python -m pip install -r requirements.txt
+
+# Test that it works, you should see a message describing the usage
+./buildingsch.py
+```
+
+## Development
+### Generate Schematron
+First create a CSV file that meets the required structure:
+```
+phase title,phase see,pattern title,pattern see,rule title,rule context,assert test,assert description,assert severity,notes
+```
+See the CSV files in this repo for examples.
+
+Hierarchy is implied by the lack of text in a column. If no phase data is added to a row, it's considered to be the same phase as the row above. If no pattern data is present, it's assumed to be the same pattern as above. If no rule context is given, it's assumed to be the same as the one above.
+
+The generator expects a "golden" xml file which should pass the validation. This is used to make sure all rules are applied (schematron will skip rules if the rule context doesn't match or if it only matches nodes that have already been matched within that pattern). If no golden file is provided no rule context checks will be made.
```

Comment:
This is going to get nit-picky, but when I first saw "golden" I thought of this [Golden File](https://github.com/BuildingSync/schema/blob/develop/examples/Golden%20Test%20File.xml), which, bc it's related to BSync, could get confusing.  Maybe we can refer to them as: 'exemplary'?

Reply:
changed to exemplary